en/extend_your_application/README.md

```diff
@@ -30,7 +30,7 @@ We will start with adding a link inside `blog/templates/blog/post_list.html` fil
 {% endblock %}
 ```
 
-{% raw %}We want to have a link from a post's title in the post list to the post's detail page. Let's change `<h1><a href="">{{ post.title }}</a></h1>` so that it links to the post's detail page:{% endraw %}
+{% raw %}We want to have a link from a post's title in the post list to the post's detail page. Let's change `<h2><a href="">{{ post.title }}</a></h2>` so that it links to the post's detail page:{% endraw %}
 
 {% filename %}{{ warning_icon }} blog/templates/blog/post_list.html{% endfilename %}
 ```html
```

Comment:
I think in the code block below, it should be changed to `<h2>`, too. There's no reason to make the blog title a top-level title when we already use the blog name as the top-level title.

Reply:
Exactly! It should either be all h1 or h2, preferably h2.

Reply:
This has been addressed by e15cf44a96b19b7cfc3af42470cfcf5f34ec3c38